StabQuest/Character.cs

```diff
@@ -76,16 +78,34 @@ public void Print()
 
         public int Level { get; set; }
 
-        public int CurrentExperience { get; set; }
-        public int ExperienceForNextLevel { get; set; }
+        public int CurrentExperience {
+            get { return _currentExperience; }
+            set {
+                _currentExperience  = value;
+
+                if(_currentExperience >= ExperienceForNextLevel)
+                {
+                    LevelUp();
+                }
+            } 
+        }
+
+        public void LevelUp()
+        {
+            Level++;
+            UnspentSkillPoints += (int)((10 + Level) / 10) + 1;
```

Comment:
Reduntant cast since var is already int :)
```suggestion
            UnspentSkillPoints += ((10 + Level) / 10) + 1;
```